src/main/java/org/gridsuite/modification/server/modifications/BatteryModification.java

```diff
@@ -171,7 +172,8 @@ public static ReportNode modifyBatteryActivePowerControlAttributes(AttributeModi
                                                                        ReportNode subReportNodeSetpoints) {
         ActivePowerControl<Battery> activePowerControl = battery.getExtension(ActivePowerControl.class);
         ActivePowerControlAdder<Battery> activePowerControlAdder = battery.newExtension(ActivePowerControlAdder.class);
-        return ModificationUtils.getInstance().modifyActivePowerControlAttributes(activePowerControl, activePowerControlAdder, participate, droop, subReportNode, subReportNodeSetpoints);
+        return ModificationUtils.getInstance().modifyActivePowerControlAttributes(activePowerControl, activePowerControlAdder,
+            participate, droop, subReportNode, subReportNodeSetpoints, CREATE_BATTERY_ERROR, "Battery '" + battery.getId() + "' : ");
```

Comment:
MODIFY_BATTERY_ERROR not CREATE_BATTERY_ERROR
and you can create const variable errorMessage for CREATE_BATTERY_ERROR, "Battery '" + battery.getId() + "' : "